[DOC] Normalizers (#8192)
opensearch-project/documentation-website: _analyzers/index.md

}
```

## Normalizers

> While tokenization divides text into individual terms, it doesn’t address variations in token forms. Normalization resolves these issues by converting tokens into a standard format. This ensures that similar terms are matched appropriately, even if they aren’t identical.

Reviewer: ```suggestion
Tokenization divides text into individual terms, but it does not address variations in token forms. Normalization resolves these issues by converting tokens into a standard format. This ensures that similar terms are matched appropriately, even if they are not identical.
```

> ### Normalization techniques

Reviewer: ```suggestion
### Normalization techniques

The following normalization techniques can help address variations in token forms:
```

Reviewer: @leanneeliatra I added an intro sentence.

}
```

## Normalizers
Tokenization divides text into individual terms, but it does not address variations in token forms. Normalization resolves these issues by converting tokens into a standard format. This ensures that similar terms are matched appropriately, even if they are not identical.

### Normalization techniques

The following normalization techniques can help address variations in token forms:
1. **Case normalization**: Converts all tokens to lowercase to ensure case-insensitive matching. For example, "Hello" is normalized to "hello".

2. **Stemming**: Reduces words to their root form. For instance, "cars" is stemmed to "car", and "running" is normalized to "run".

3. **Synonym handling:** Treats synonyms as equivalent. For example, "jogging" and "running" can be indexed under a common term, such as "run".

### Normalization

> A search for `Hello` will match documents containing `hello` due to case normalization.

Reviewer 2: ```suggestion
A search for `Hello` will match documents containing `hello` because of case normalization.
```

> Searching for `cars` will also match documents with `car` because of stemming.

Reviewer 2: ```suggestion
A search for `cars` will also match documents containing `car` because of stemming.
```

> A query for `running` can retrieve documents containing `jogging` through synonym handling.

Reviewer 2: ```suggestion
A query for `running` can retrieve documents containing `jogging` using synonym handling.
```
